src/Storages/System/StorageSystemColumns.cpp

```diff
@@ -148,7 +149,19 @@ class ColumnsSource : public ISource
                     continue;
                 }
 
-                auto metadata_snapshot = storage->getInMemoryMetadataPtr();
+                StorageMetadataPtr metadata_snapshot;
+                /// Can throw UNKNOWN_DATABASE/UNKNOWN_TABLE in case of Alias table
+                try
+                {
+                    metadata_snapshot = storage->getInMemoryMetadataPtr();
+                }
+                catch (const Exception & e)
+                {
+                    if (e.code() != ErrorCodes::UNKNOWN_DATABASE && e.code() != ErrorCodes::UNKNOWN_TABLE)
+                        throw;
+                    metadata_snapshot = std::make_shared<StorageInMemoryMetadata>();
```

Comment:
I meant remove all the try/catch inside StorageAlias and instead do it here, and only if engine is Alias, and in case of it throws - just skip the table

Reply:
I'm suggesting to skip this table (`continue` here), instead of replacing it with dummy